gallery/lib/home.dart

```diff
@@ -0,0 +1,121 @@
+import 'package:flutter/material.dart';
+
+import 'demo.dart';
+import 'settings.dart';
+import 'splash.dart';
+
+class HomePage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('Gallery'),
+        leading: IconButton(icon: Icon(Icons.color_lens), onPressed: () {
+          Navigator.of(context).push(
+            MaterialPageRoute(
+              fullscreenDialog: true,
+              builder: (context) => SplashPage(),
+            ),
+          );
+        }),
+        actions: <Widget>[
+          IconButton(icon: Icon(Icons.settings), onPressed: () {
+            Navigator.of(context).push(
+              MaterialPageRoute(
+                fullscreenDialog: true,
+                builder: (context) => SettingsPage(),
+              ),
+            );
+          }),
+        ],
+      ),
+      body: Padding(
+        padding: const EdgeInsets.all(16.0),
+        child: ListView(
+          children: <Widget>[
+            Text('Gallery', style: Theme.of(context).textTheme.headline,),
+            Container(
+              height: 200,
+              child: ListView(
+                scrollDirection: Axis.horizontal,
+                children: <Widget>[
+                  CarouselCard(title: 'Shrine'),
+                  CarouselCard(title: 'Rally'),
+                  CarouselCard(title: 'Crane'),
+                ],
+              ),
+            ),
+            Text('Categories', style: Theme.of(context).textTheme.headline,),
```

Comment:
probably don't need the trailing comma, but if you leave it, make sure you run `dartfmt` so this fixes itself.

Reply:
Done